src/main/webapp/resources/js/modules/cart/irida.cart.js

```diff
@@ -1,37 +0,0 @@
-import angular from "angular";
-import { CART } from "../../utilities/events-utilities";
-import { emptyCart, removeSample } from "../../apis/cart/cart";
-
-function CartService(scope, $http) {
```

Comment:
I see the `removeSample` function reimplemented, what about the other service functions in here?  I'm assuming they're already done somewhere else?

Reply:
So the answer to all your questions is that these functions had already been re-written for updates to other features (mostly the new cart).  The only functionality that was really still used was remove sample from cart I just refactored to use the new one.